### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by chrissyalbert**
Please check feature/footer code. The links were a challenge and my solution might not be the best. Any ideas?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,10 @@
     "build": "vue-cli-service build"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
```
Comment (rvehall) [inline package.json]: We're using Material Icons - they're imported with Vuetify. Please update to remove the fontawesome packages and use the Material Icons. Thanks!

File: src/components/Footer.vue [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    data: () => ({
+      links: [
+        {name: 'Home', href: "/"},
+        {name: 'About', href: "/about"}
+      ],
+    }),
+  }
+</script>
+
+<!-- {name: 'GitHub', href: "https://github.com/Women-Who-Code-Phoenix/lah"} -->
```
Comment (rvehall) [inline src/components/Footer.vue]: You can remove this commented code.

**Review by rvehall — CHANGES_REQUESTED** [review]
Can you add "Closes #12" to your comment in order to associate this with the ticket. Thanks!

**Comment by rvehall** [comment]
Also, this is pointing to merge into `master`. Please point it to merge into `development`. Thanks!

File: src/components/Footer.vue (hunk 2 above)
Comment (chrissyalbert) [reply src/components/Footer.vue]: done

File: package.json (hunk 1 above)
Comment (chrissyalbert) [reply package.json]: I couldn't find the exact icon with external link--this one is close.

**Comment by chrissyalbert** [comment]
"Closes #12"

**Pull request closed without merging**